Dockerfile_ubuntu_1804_cuda_11_3_cudnn_8_torchgeo

```diff
@@ -0,0 +1,164 @@
+FROM nvidia/cuda:11.3.1-cudnn8-devel-ubuntu18.04
```

Comment:
I'm guessing this was added by accident?

Reply:
yes i need to roll this back ..and add it as a separate PR head..i wanted to create a different header for this PR but it got pushed to SWI PR

Reply:
I recommend creating a separate branch for each PR, that way changes don't start to end up in multiple PRs

Reply:
thanks